samples/sample/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     annotationProcessor project(':compiler')
     implementation project(':samples:sample-lib')
     implementation project(':lib')
+    implementation deps.dagger
```

Comment:
This is what we want to avoid by keeping Dagger as `api`. Even though `:sample` does not import Dagger in any committed code, Motif generates Dagger code which requires Dagger to be on the classpath.